spylon_kernel/init_spark_magic.py

```diff
@@ -21,33 +32,42 @@ def __init__(self, kernel):
         self.log = logging.Logger("InitSparkMagic")
 
     def cell_init_spark(self):
-        """
-        %%init_spark - start up a spark context with a custom configuration
+        """Starts a SparkContext with a custom configuration defined
+        using Python code.
 
-        Example:
-            %%init_spark
-            application_name = 'My Fancy App'
-            launcher.jars = ["file://some/jar.jar"]
-            launcher.master = "local[4]"
-            launcher.conf.spark.executor.cores = 8
+        Includes a `spylon.spark.launcher.SparkConfiguration` instance
+        in the variable `launcher`. Looks for an `application_name`
+        variable to use as the name of the Spark session.
 
-        This will evaluate the launcher args using spylon.
+        Example
+        -------
+        %%init_spark
+        application_name = "My Fancy App"
+        launcher.jars = ["file://some/jar.jar"]
+        launcher.master = "local[4]"
```

Comment:
Spark syntax for how many cores to use. `local[4]` says "Please use 4 of my cores for the application master." `local[*]` says "TAKE ALL MY CORES PLZ!"